documentation/src/main/markdown/runningdetect/threatintelscan.md

```diff
@@ -0,0 +1,44 @@
+# [threat_intel] Scan
+
+[threat_intel] Scans are a way of running binary file analysis that provides malware warnings, with a risk analysis level applied, for open source and commercial software.
+
+[company_name] [solution_name] will accept a user provided local file path to a binary file for [threat_intel] Scan.
```

Comment:
Might it be worth mentioning that this file could be a compressed file, like a tar or zip, that contains many files?